control/trajectory_follower_nodes/src/lateral_controller_node.cpp

```diff
@@ -146,20 +146,20 @@ LateralController::LateralController(const rclcpp::NodeOptions & node_options)
 
   m_pub_ctrl_cmd =
     create_publisher<autoware_auto_control_msgs::msg::AckermannLateralCommand>(
-    "output/lateral/control_cmd", 1);
+    "~/output/lateral/control_cmd", 1);
```

Comment:
If we use the private namespace (with `~`), the topic will look like `/namespace/lateral_controller_node/output/lateral/control_cmd`. I think in that case it is not necessary to specify `lateral`.
```suggestion
    "~/output/control_cmd", 1);
```
Same suggestion for the other topics and for the longitudinal controller node.

Reply:
I think your suggestion looks nice.
I'll change them.